pages/first-project.mdx

```diff
@@ -0,0 +1,362 @@
+import { Callout, Cards, Steps, Tabs } from "nextra/components";
+
+# **Getting Started: Create Your First Project**
+
+In this guide, we’ll walk you through building your first project with Unbody, step by step.
+
+<Callout type="info" emoji="⚠️">
```

Comment:
this is a warning emoji while the type is info! so the emoji should also be info